pkg/proxy/iptables/proxier.go

```diff
@@ -673,6 +673,9 @@ func (proxier *Proxier) syncProxyRules() {
 		if svcInfo, ok := proxier.serviceMap[svcPortName]; ok && svcInfo != nil && svcInfo.GetProtocol() == v1.ProtocolUDP {
 			klog.V(2).Infof("Stale udp service %v -> %s", svcPortName, svcInfo.ClusterIPString())
 			staleServices.Insert(svcInfo.ClusterIPString())
+			for _, extIP := range svcInfo.ExternalIPStrings() {
```

Comment:
Should we catch status.ingress.ip too?

Reply:
@thockin yes we should catch status.ingress.ip also. I will submit another PR